srcpkgs/cockatrice/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'cockatrice'
+pkgname=cockatrice
+version=2.8.0
+revision=1
+_reldate=2021-01-26
+build_style=cmake
+hostmakedepends="protobuf protobuf-devel"
```

Comment:
only `protobuf` should go in hostmakedepends because it needs to run `protoc` from that package, right? `protobuf-devel` needs to go in makedepends because it's for compiling against the protobuf library